### DISCUSSION TITLE
Why does this cost money?????

### DISCUSSION THREAD

**Discussion by henryruss2** [plausible/analytics#853] [Anything Else]
This claims to be open source, but it costs money! I won't use this unless it is truly open source.

**Comment by Vigasaurus** [comment]
https://plausible.io/docs/self-hosting 🤨

**Reply by metmarkosaric** [reply]
Free and open source software doesn't have anything to do with the price of the product. You can charge for it or not - it doesn't matter. Even with a price it can still be free in the sense of the word as they meant it (freedom). But you can still get Plausible free (as in beer) if you want to.
